fail2ban/protocol.py

```diff
@@ -26,6 +26,9 @@
 
 import textwrap
 
+def output(s):
+	print(s)
```

Comment:
So what is the purpose of having it a function? To ease mocking in the tests or ...? Please describe in the docstring